taskcluster_util/taskcluster_login.py

```diff
@@ -90,6 +90,23 @@ def cli(self):
         return self
 
     class ServerHandler(BaseHTTPRequestHandler):
+        def credentials_file_handler(self, content):
```

Comment:
Nit: if the file hanlder is not a class, I'd put a verb in this function name. For example: `write_to_crendentials_file()`